consensus/entry_point_selector/tests/test_entry_point_selector.c

```diff
@@ -12,57 +12,57 @@
 #include "consensus/entry_point_selector/entry_point_selector.h"
 #include "consensus/milestone/milestone.h"
 #include "consensus/tangle/tangle.h"
+#include "consensus/test_definitions/defs.h"
 #include "utils/files.h"
 
 static entry_point_selector_t eps;
 static tangle_t tangle;
 static milestone_t milestone;
+connection_config_t config;
+
+// gdb --args ./test_cw_ratings_dfs 1
```

Comment:
this comment is not relevant anymore

Reply:
It's still, until the path thingy is resolved